kyo-zio/shared/src/main/scala/kyo/ZIOs.scala

```diff
@@ -67,4 +73,28 @@ object ZIOs:
         }
     end run
 
+    private object internal:
+        extension [E, A](exit: zio.Exit[E, A])
```

Comment:
Could you introduce tests for these extensions? I wonder if we should make them public, I imagine they could be useful in user code as well